src/Helper/OutputResultConverter.php

```diff
@@ -48,12 +51,22 @@ public static function convertOutputsToResult(array $outputs): JobResult
                     $inputTables->addTable(self::convertTableInfoToTableResult($tableInfo));
                 }
             }
+            $uploadedArtifactOutput = $output->getArtifactUploaded();
+            if ($uploadedArtifactOutput) {
+                $artifacts->setUploaded($uploadedArtifactOutput);
```

Comment:
co  $artifacts->setUploaded($uploadedArtifactOutput ? []);

Reply:
a proč se to vlastně nenastavuje stejně jako ty downloadedArtifacts? Takhle tam zůstane poslední output ne?

Reply:
Ja som to tak spravil schvalne, pretoze ten `uploaded` bude len jeden subor (z hlavneho imagu), ale teraz mi docvaklo, ze ked su tam configRows, tak aj tych `uploaded` bude viac vlastne.
Takze to spravim rovnako ako tie downloaded predsa len.